fdbserver/RestoreApplier.actor.cpp

```diff
@@ -381,13 +384,25 @@ ACTOR static Future<Void> applyStagingKeysBatch(std::map<Key, StagingKey>::itera
 			while (iter != end) {
 				if (iter->second.type == MutationRef::SetValue) {
 					tr->set(iter->second.key, iter->second.val);
+					TraceEvent(SevFRMutationInfo, "FastRestoreApplierPhaseApplyStagingKeysBatch", applierID)
```

Comment:
Right now, SevFRMutationInfo will not create any event because it is defined as SevVerbose.

I plan to use the debugFRMutation() or a similar approach that allows DBA to control when to print out which mutation from the backup data.